### DISCUSSION TITLE
Hundreds of repeat commits for yargs-parser updates

### DISCUSSION THREAD

**Discussion by tschaub** [renovatebot/renovate#9837] [Request Help]
It looks like @renovate-bot is pushing hundreds of commits to the [jsdoc/jsdoc](https://github.com/jsdoc/jsdoc) repo that [add](https://github.com/jsdoc/jsdoc/commit/a8a9573b7f2b040a65612ffdcaff8aaa86cfbb2c) and then [remove](https://github.com/jsdoc/jsdoc/commit/82fcf02a981835f0d6cc8b92669936ed3dcb29a7) the same two lines from `package-lock.json` related to a `yargs-parser` package.  See https://github.com/jsdoc/jsdoc/commits?author=renovate-bot

It looks like these are pointless commits and it is unclear if they will ever stop.  Any ideas whether this is [a configuration](https://github.com/jsdoc/jsdoc/blob/113f4acba015dd3cb52f15a6d19b56530e955889/.renovaterc.json) issue or an issue with the bot itself?

**Comment by rarkins** [comment]
The root cause is the same as this: https://github.com/yargs/yargs-parser/pull/363#issuecomment-830646158

i.e. GitHub's vulnerability metadata is wrong and flags `5.0.1` as vulnerable to there's a loop of fixing.

Obviously we don't want that to happen and really sorry you're experiencing it. I would advise disabling automerge temporarily until I can figure out a workaround.

**Reply by rarkins** [reply]
I am adding a hardcoded override for this incorrect alert and will try to get it into the app within an hour or too.

**Reply by rarkins** [reply]
Now done. Please let me know if any further commits happen

**Reply by tschaub** [reply]
Thanks, @rarkins.

(For folks who might read this later, the commits mentioned above have been removed from the default branch, so listing commits by the @renovate-bot author will no longer show the large batch of commits that motivated this discussion.)
